sections/introduction.tex

```diff
@@ -1,17 +1,16 @@
 \section{Introduction}\label{sec:introduction}
-% Intro to meta data
-Data is everywhere and it is used in many different scenarios and use cases. 
-When referring to a dataset, we talk about a dataset including metadata information, which are non-essential data features.
-One example of this could be the publication date of a text document. 
-In most contexts, it is not needed to use and understand the document; however, it provides more context to the main content of the document: the text.
-However, a lot of metadata is not used, either because it is not useful or we do not know how to use it properly.
+% Intro to recommendation and techniques
+Broadcasting information around the world is a crucial skill, that is used every day, as news articles, television and newspapers supply people with the latest information.
+Searching and categorizing the news is becoming a bigger problem, since news are created every day, if not every hour.
+As the internet, becomes more personalized, like targeted advertisement, recommendation is seeing more use in applications, such as Netflix and Amazon.
+Within recommendation, there are two main machine learning techniques: Collaborative filtering and Content based filtering.
+These two techniques are based on assumption that similar users like similar items (collaborative) or one user likes certain types items(content based).
+The recommendation task can be very specific depending on the dataset that is used.
+We focus on creating a better environment for recommendation of news articles.
```

Comment:
```suggestion
Broadcasting information around the world is a crucial skill, that is used every day, like news articles, television and newspapers supply people with the latest information.
Searching and categorizing the news is becoming a bigger problem since the news is created every day, if not every hour.
As the internet, becomes more personalized, like a targeted advertisement, recommendation is seeing more use in applications, such as Netflix and Amazon.
Within recommendation, there are two main machine learning techniques: Collaborative filtering and Content-based filtering.
These two techniques are based on assumption that similar users like similar items (collaborative) or one user likes certain types of items(content-based).
The recommendation task can be very specific depending on the dataset that is used.
We focus on creating a better environment for the recommendation of news articles.
```